fake_switches/netconf/__init__.py

```diff
@@ -109,11 +110,32 @@ def __init__(self, msg, severity="error", err_type=None, tag=None, info=None, pa
         self.info = info
         self.path = path
 
+    def to_dict(self):
+        return error_to_rpcerror_dict(self)
+
+    def to_etree(self):
+        return dict_2_etree(self.to_dict())
+
```

Comment:
let it breathe

Reply:
breath has been given

https://cdn.movieweb.com/img.news.tops/NEfArFf7LqO8il_3_b/Frozen-Movie-Lawsuit-Disney-Idina-Menzel-Demi-Lovato.jpg